ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/results/EngineGetPayloadResultV2.java

```diff
@@ -39,8 +41,9 @@ public EngineGetPayloadResultV2(
       final BlockHeader header,
       final List<String> transactions,
       final Optional<List<Withdrawal>> withdrawals,
+      final Optional<List<Deposit>> deposits,
```

Comment:
It's not part of this PR so I couldn't add a comment directly, but it looks like this replaces the `EngineGetPayloadResultV6110` that was added in the previous PR and `EngineGetPayloadResultV6110` can now be removed.

Reply:
Yea it fell off the radar